qcodes_contrib_drivers/drivers/Bilt/ITest.py

```diff
@@ -118,7 +119,7 @@ def __init__(self, parent: Instrument,
                            get_parser=float,
                            set_cmd=None,
                            vals=vals.Numbers(1e-3, 10),
-                           initial_value=1)
+                           initial_value=1e-3)
```

Comment:
Is this change intentional ? Does not seem related

Reply:
Sorry some how missed this

Reply:
no problem, it is indeed not related to the fix but nice to change the default value.